changelog/7200.bugfix.md

```diff
@@ -0,0 +1 @@
+Fix a bug because of which only one retrieval intent was present in `all_retrieval_intent` key of the output of `ResponseSelector` even if there where multiple retrieval intents present in the training data.
```

Comment:
```suggestion
Fix a bug because of which only one retrieval intent was present in `all_retrieval_intent` key of the output of `ResponseSelector` even if there were multiple retrieval intents present in the training data.
```